Task_DEV-2/Calculator.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task_DEV_2
+{
+    // Class which gets checed string and do all operation 
+    class Calculator
+    {
+        private string inputString;
+        private string expressionOperator = "*-+/";
+
+        // constructor 
+        public Calculator(string inputString)
+        {
+            this.inputString = inputString;
+        }
+
+        // method which make oeration with 2 variables
+        // and return result
+        private double[] ResultOfOperation(double[] stack, ref bool[] isNumber,
+            int stackCount, string operation)
+        {
+            double firstVariable = 0;
+            double secondVariable = 0;
+            int i = 0;
+            while (i < stackCount)
+            {
+                i++;
+                if (isNumber[stackCount - i])
+                {
+                    secondVariable = stack[stackCount - i];
+                    isNumber[stackCount - i] = false;
+                    stack[stackCount - i] = 0;
+                    while (i < stackCount)
+                    {
+                        i++;
+                        if (isNumber[stackCount - i])
+                        {
+                            firstVariable = stack[stackCount - i];
+                            isNumber[stackCount - i] = false;
+                            stack[stackCount - i] = 0;
+                            break;
+                        }
+                    }
+                    isNumber[stackCount] = true;
+                    break;
+                }
+
+            }
+            switch (operation)
+            {
+                case "*":
+                    stack[stackCount] = firstVariable * secondVariable;
+                    return stack;
+                case "/":
+                    stack[stackCount] = firstVariable / secondVariable;
+                    return stack;
+                case "+":
+                    stack[stackCount] = firstVariable + secondVariable;
+                    return stack;
+                case "-":
+                    stack[stackCount] = firstVariable - secondVariable;
+                    return stack;
+                default:
+                    return stack;
+            }
+
+        }
+
+        // method which gets string in postfix format and do all operation
+        // and return result
+        public double Calculate()
```

Comment:
Can't understand nothing, what is stack? what is Infix and postfix conversion, what  for?